workflows/employment_verification.go

```diff
@@ -0,0 +1,86 @@
+package workflows
+
+import (
+	"math/rand"
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+
+	"github.com/temporalio/background-checks/config"
+	"github.com/temporalio/background-checks/signals"
+	"github.com/temporalio/background-checks/types"
+)
+
+func chooseResearcher(ctx workflow.Context, input types.EmploymentVerificationWorkflowInput) (string, error) {
```

Comment:
Unnecessary `ctx` param

Reply:
It was, but it's now used by a side effect call ;)